Defs/Ammo/Neolithic/SlingBullet.xml

```diff
@@ -1,213 +1,173 @@
+    <statBases>
+      <Mass>0.03</Mass>
+      <Bulk>0.07</Bulk>
+    </statBases>
+    <thingCategories>
+      <li>AmmoSlingBullet</li>
+    </thingCategories>
+    <stackLimit>5000</stackLimit>
+    <tradeTags>
+      <li>CE_AutoEnableTrade</li>
+      <li>CE_AutoEnableCrafting_FueledSmithy</li>
+      <li>CE_AutoEnableCrafting_ElectricSmithy</li>
+    </tradeTags>
+  </ThingDef>
+
+  <ThingDef Class="CombatExtended.AmmoDef" ParentName="AmmoSlingBulletBase">
+    <defName>Ammo_SlingBullet_Stone</defName>
+    <label>sling bullet (stone)</label>
+    <graphicData>
+      <texPath>Things/Ammo/Neolithic/SlingBullet/Stone</texPath>
+      <graphicClass>Graphic_StackCount</graphicClass>
+    </graphicData>
+    <statBases>
+      <MarketValue>0.1</MarketValue>
+    </statBases>
+		<ammoClass>StoneSlingBullet</ammoClass>
+    <tradeTags>
+      <li>CE_AutoEnableCrafting_CraftingSpot</li>
+    </tradeTags>
+  </ThingDef>
+
+  <ThingDef Class="CombatExtended.AmmoDef" ParentName="AmmoSlingBulletBase">
+    <defName>Ammo_SlingBullet_Steel</defName>
+    <label>sling bullet (steel)</label>
+    <graphicData>
+      <texPath>Things/Ammo/Neolithic/SlingBullet/Steel</texPath>
+      <graphicClass>Graphic_StackCount</graphicClass>
+    </graphicData>
+    <statBases>
+      <Mass>0.085</Mass>	
+      <MarketValue>0.19</MarketValue>
+    </statBases>
+		<ammoClass>SteelSlingBullet</ammoClass>
+  </ThingDef>
+	
+	<!-- ================== Projectiles ================== -->
+
+	<ThingDef Class="CombatExtended.AmmoDef" Name="BaseSlingBullet" ParentName="BaseBullet" Abstract="true">
+		<graphicData>
+			<texPath>Things/Mote/Stone</texPath>
+			<graphicClass>Graphic_Single</graphicClass>
+		</graphicData>
+		<projectile Class="CombatExtended.ProjectilePropertiesCE">
+			<damageDef>Blunt</damageDef>
+			<speed>9</speed>
+		</projectile>
+	</ThingDef>
+
+	<!-- The amount of arrows usable over the ammo lifetime for each resource cost is equal to
+		[r / (1 - x)];
+			x = preExplosionSpawnChance,
+			r = resource cost. -->
+
+	<ThingDef Class="CombatExtended.AmmoDef" ParentName="BaseSlingBullet">
+		<defName>Projectile_SlingBullet_Stone</defName>
+		<label>sling bullet (stone)</label>
+		<graphicData>
+			<texPath>Projectiles/Sling/Stone</texPath>
+			<graphicClass>Graphic_Single</graphicClass>
+		</graphicData>		
+		<projectile Class="CombatExtended.ProjectilePropertiesCE">
+			<damageAmountBase>4</damageAmountBase>
+			<armorPenetrationBlunt>0.6</armorPenetrationBlunt>
+			<preExplosionSpawnChance>0.166</preExplosionSpawnChance>	<!-- 11.99 stones per resource -->
+			<preExplosionSpawnThingDef>Ammo_SlingBullet_Stone</preExplosionSpawnThingDef>
+		</projectile>
+	</ThingDef>
+
+	<ThingDef Class="CombatExtended.AmmoDef" ParentName="BaseSlingBullet">
+		<defName>Projectile_SlingBullet_Steel</defName>
+		<label>sling bullet (steel)</label>
+		<graphicData>
+			<texPath>Projectiles/Sling/Steel</texPath>

```

Comment:
Missing texture at this path.

Reply:
Uploaded new projectile texture, update path.